website/content/docs/upgrading/upgrade-to-1.15.x.mdx

```diff
@@ -40,3 +40,11 @@ To continue measuring `vault.rollback.attempts.{MOUNTPOINT}` and
 metrics in the `telemetry` stanza of your Vault configuration with the
 [`add_mount_point_rollback_metrics`](/vault/docs/configuration/telemetry#add_mount_point_rollback_metrics)
 option.
+
+## Application of Sentinel Role Governing Policies (RGPs) via identity groups
+
+As of versions 1.15, 1.14.4, and 1.13.8, [the Sentinel RGPSs derived from membership in identity groups apply
```

Comment:
```suggestion
As of versions `1.15.0`, `1.14.4`, and `1.13.8`, [the Sentinel RGPSs derived from membership in identity groups apply
```

🤔 up to you if you think `this` looks more obvious or not 😄 

Added the `.0` on the version just to keep it the same as the way its formatted in the [other PR](https://github.com/hashicorp/vault/pull/23292)

Reply:
Yeah, seems like a good shout to me. Thanks, Pete!